packages/build-tools/src/ios/tvos.ts

```diff
@@ -0,0 +1,30 @@
+import { Ios } from '@expo/eas-build-job';
+import { IOSConfig } from '@expo/config-plugins';
+
+import { BuildContext } from '../context';
+
+// Functions specific to Apple TV support should be added here
+
+/**
+ * Use XcodeUtils to check if a build configuration is for Apple TV and not iOS
+ *
+ * @param ctx The build context
+ * @returns true if this is an Apple TV configuration, false otherwise
+ */
+export async function isTVOS(ctx: BuildContext<Ios.Job>): Promise<boolean> {
+  if (!ctx.job.scheme) {
+    return false;
+  }
+  const project = IOSConfig.XcodeUtils.getPbxproj(ctx.reactNativeProjectDirectory);
+
+  const targetName = await IOSConfig.BuildScheme.getApplicationTargetNameForSchemeAsync(
+    ctx.reactNativeProjectDirectory,
+    ctx.job.scheme || ''
```

Comment:
Sorry, I should have been more clear in https://github.com/expo/eas-build/pull/129#discussion_r938547526 I meant that buildConfiguration is passed in job and as an argument but those are not the same values,

You need to use values resolved here https://github.com/expo/eas-build/blob/main/packages/build-tools/src/builders/ios.ts#L92

Reply:
Thanks for the clarification! I think I'm using the `resolveBuildConfiguration()` method correctly now.